feat(changelog): kubernetes-removed-container-storage-interface-csi-driv 2025-01-15 (#4223)
scaleway/docs-content: changelog/january2025/2025-01-15-kubernetes-removed-container-storage-interface-csi-driv.mdx

---
title: Container Storage Interface (CSI) driver v.0.2 no longer supported
status: removed
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #k8s channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2025-01-15
category: containers
product: kubernetes
---

> The Scaleway Block Volume Container Storage Interface (CSI) driver v.0.2 will no longer be supported by 31/01/2025.
> 
> Users can upgrade to the latest CSI driver (v0.3) now.
> 
> Learn more [here](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/managing-storage).

Reviewer: ```suggestion
The Scaleway Block Volume Container Storage Interface (CSI) driver **v0.2** will reach its end of support on **January 31, 2025**.  

To ensure continued compatibility, **we recommend upgrading to the latest version (v0.3)** as soon as possible.  

Refer to our dedicated documentation and learn how to upgrade: [Managing Block Storage volumes with Scaleway CSI](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/managing-storage/)

```

---

> title: Container Storage Interface (CSI) driver v.0.2 no longer supported

Reviewer 2: ```suggestion
title: Container Storage Interface (CSI) driver v.0.2 is no longer supported
```

status: removed
author:
    fullname: 'Join the #k8s channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2025-01-15
category: containers
product: kubernetes
---

The Scaleway Block Volume Container Storage Interface (CSI) driver **v0.2** will reach its end of support on **January 31, 2025**.  

To ensure continued compatibility, **we recommend upgrading to the latest version (v0.3)** as soon as possible.  

> Refer to our dedicated documentation and learn how to upgrade: [Managing Block Storage volumes with Scaleway CSI](https://www.scaleway.com/en/docs/containers/kubernetes/api-cli/managing-storage/)
> 

Reviewer 2: ```suggestion
Refer to our dedicated documentation and learn how to upgrade: [Managing Block Storage volumes with Scaleway CSI](/containers/kubernetes/api-cli/managing-storage/)
```
